backend/app/api/endpoints/internal/model_config.py

```
@@ -0,0 +1,146 @@
+        HTTPException 400: structurally invalid model_ref, or model does not
+            declare the required capability for ``media_type``.
+    """
+    try:
+        cfg = resolve_model_config_by_ref(
+            db,
+            model_ref=request.model_ref,
+            user_id=request.uploader_id,
+            user_name=request.uploader_name,
+        )
+    except ModelRefResolutionError as exc:
+        raise HTTPException(status_code=400, detail=f"{exc.code}: {exc}")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)